src/main/kotlin/org/stellar/walletsdk/util/SetRecoveryMethods.kt

```diff
@@ -6,6 +6,26 @@ import kotlinx.coroutines.async
 import okhttp3.OkHttpClient
 import org.stellar.walletsdk.*
 
+/**
+ * Helper to register account with a recovery server using
+ * [SEP-30](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0030.md) and
+ * [SEP-10](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0010.md).
+ *
+ * @param endpoint Recovery server's endpoint URL
```

Comment:
Call this `recoveryServerEndpoint` to be more descriptive?

Reply:
All of those params are for the recovery server. Should we prefix all of them with `recoveryServer`? It seemed a bit redundant to repeat the "recovery server" in them. Maybe it would help with clarity? What do you think, @piyalbasu ?

Reply:
Oh gotcha. Yeah, let's not do that. Hmmmm, I was just trying to find a way to make `endpoint` more descriptive, but I guess the SEP doesn't really provide a better naming convention. Ignore my comment - the @param description should hopefully be enough